src/data/downloadMetadata.ts

```diff
@@ -1983,17 +1983,17 @@ export const dataPages: DownloadMetadata = {
     layerId: 0,
   },
   'Utah UTA Light Rail Stations': {
```

Comment:
**issue**: the key name 'Utah UTA Light Rain Stations' needs to match the name property